src/internal/sftpgo/sftpgo.go

```diff
@@ -32,35 +32,29 @@ func rndPass() string {
 	return hex.EncodeToString(data)
 }
 
-type token struct {
-	AccessToken string    `json:"access_token"`
-	ExpiresAt   time.Time `json:"expires_at"`
-}
-
 // API is used to send API requests to the SFTPGo daemon
 type API struct {
 	m       sync.Mutex
```

Comment:
Remove `m` - the mutex is no longer used

Reply:
Thanks for catching this. Removing.